service/controller/resource/encryptionensurer/create.go

```diff
@@ -16,44 +13,11 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	if err != nil {
 		return microerror.Mask(err)
 	}
-	cc, err := controllercontext.FromContext(ctx)
-	if err != nil {
-		return microerror.Mask(err)
-	}
 
-	// The tccpencryption resource is reconciled upon the AWSCluster CR which
-	// defines the TCCP Cloud Formation stack. With the provider specific Cluster
-	// CR we ensure the encryption key. Note that the TCNP stack for the Node
-	// Pools also needs the encryption key.
 	err = r.encrypter.EnsureCreatedEncryptionKey(ctx, cr)
 	if err != nil {
 		return microerror.Mask(err)
 	}
 
-	// For some obscure reasons the encryption key is not immediately available
-	// when creating it. On each cluster creation we saw the retry resource
-	// kicking in once because of a not found error. To prevent the error, instead
-	// we backoff silently upfront where we know we have to.
-	{
-		var encryptionKey string
-
-		o := func() error {
-			encryptionKey, err = r.encrypter.EncryptionKey(ctx, cr)
-			if err != nil {
-				return microerror.Mask(err)
-			}
-
-			return nil
-		}
-		b := backoff.NewMaxRetries(3, 1*time.Second)
-
-		err := backoff.Retry(o, b)
-		if err != nil {
-			return microerror.Mask(err)
-		}
-
-		cc.Status.TenantCluster.Encryption.Key = encryptionKey
```

Comment:
This is done in the `encryptionsearcher` resource, so we have a nicer separation and more clear building blocks with that separated resource design. Further, with HA Masters the encryption key is not used anymore for anything since all the relevant logic moved to the TCCPN stack, where the master ASG is now managed.